.eslintrc.js

```diff
@@ -61,6 +61,7 @@ module.exports = {
     'no-useless-escape': 'off',
     'no-console': 'error',
     'jsdoc/check-types': 'off',
+    'jsdoc/no-defaults': 'off',
```

Comment:
This caused a huge diff in docs of various files when autofixer was applied, so I allowed the default since it's widely used across the project